server/bound_queries.cc

```diff
@@ -1449,6 +1449,13 @@ execute(RowProcessor processor,
     typedef std::map<RowKey, GroupMapValue> GroupByMapType;
     std::vector<GroupByMapType> accum(numBuckets);
 
+    for (const auto & c: select.clauses) {
+        if (typeid(*c.get()) == typeid(WildcardExpression)) {
```

Comment:
Also, a test based upon dynamic_cast is also more reliable as it will work for derived classes.